src/videojs-hls.js

```diff
@@ -3,688 +3,518 @@
  * The main file for the HLS project.
  * License: https://github.com/videojs/videojs-contrib-hls/blob/master/LICENSE
  */
+ /*jshint -W092 */
+ /*jshint -W062 */
 (function(window, videojs, document, undefined) {
-'use strict';
 
-var
```

Comment:
I think these are all whitespace changes. If so, they should be removed from this PR.